### ISSUE TITLE
[FEATURE] "Purge and reinstall" mod Button

### ISSUE THREAD

**Issue by OverlordZorn** [Foxlider/FASTER#197]
Often, be it due to small mod updates or alot of mod updates, it can happen, that faster does not install/update a mod to 100% completion and instead, for example, keeps an outdated modfile (or key file).

In these cases, its a lot of annoying work to:
- Delete the mod from faster
- reinstalling it
- (sometimes needing to redeploy it)
- (reassigning it to the modset)

If faster would have a new dropdown menu entry called "Purge and reinstall" or simply "reinstall", which lets faster do these steps by itself for you, that would be amazing.

TLDR:
An update button which forces a clean install of the mod.

**Comment by OverlordZorn** [comment]
@jupster @liamcannon :P

**Comment by liamcannon** [comment]
@jupster I'll do it
